docs/concepts/macros/macro_variables.md

```diff
@@ -100,3 +100,22 @@ Other macro variables:
     * 'creating' - The model tables are being created.
     * 'evaluating' - The models' logic is being evaluated.
 * @gateway - A string value that represents the name of the selected [gateway](../../guides/connections.md).
+
+## User-defined Variables
```

Comment:
Since we already have "user-defined macro variables," I think it would help people differentiate if we had another descriptor for these. 

How about "user-defined project variables" since they are a top-level key in the project config?

Reply:
Maybe "parameters" is a better term here? Wdyt?

Reply:
If we're going to use the `@var()` macro name I think it would be good to use "variables" so the connection is obvious. I'm guessing the term "vars" was chosen because that's what dbt uses? 

If we were just going for clarity, I think `@project_param()` or `@project_var()` (with yaml key of same name) would both be  good.

Reply:
Not a fan of `@project_var`. Too verbose.

Reply:
Makes sense